sass/_button.scss

```diff
@@ -0,0 +1,61 @@
+//re-useable button styling
+%circle-button {
+	@extend %circle;
+}
+%oval-button {
+    border-top: 5px solid $high-light;
+    border-bottom: 5px solid $high-light;
+    border-right: 1px solid $high-light;
+    border-left: 1px solid $high-light;
+    border-radius: 100px;
+    width: 10em;
+    margin: 0 auto;
+    padding-top: 0.7em;
+    padding-bottom: 0.7em;
+    padding-left: 2em;
+    padding-right: 2em;
+}
+%google-play-button {
+	@extend %oval-button;
+	border-top-width: .2em;
+    padding-top: 1.5em;
+    border-bottom-width: .2em;
+    padding-bottom: 1.5em;
+    padding-left: 3em;
+    padding-right: 3em;
+}
+%iOS-app-button {
+	@extend %google-play-button;
```

Comment:
Since `%iOS-app-button` _only_ extends `%google-play-button`, it would have made more sense to simply call this `%call-to-action` or `%app-button` or something a bit more general. That way you only have the one button type _and_ when you call them later on, it doesn't look redundant (`.google-play-button { @extend %google-play-button; }`).